views/admin.php.inc

```diff
@@ -1,183 +1,173 @@
+                                            if (!is_null($display_opts)) {
+                                                if (!$display_opts->show_onfrontpage) {
+                                                    $checked = true;
+                                                }
+                                            }
+                                        ?>
+                                        <input type="checkbox" id="show_onfrontpage" name="show_onfrontpage" value="1" class="checkbox show_specific" 
+                                            <?php echo ($checked)?'checked':'';?> />
+                                        <label class="option" for="show_onfrontpage">Show on front page</label>
+                                    </div>
+                                    <div class="form-item form-type-checkbox div_show_specific">
+                                        <?php
+                                            $checked = false;
+                                            if (!is_null($display_opts)) {
+                                                if (!$display_opts->show_oncategory) {
+                                                    $checked = true;
+                                                }
+                                            }
+                                        ?>
+                                        <input type="checkbox" id="show_oncategory" name="show_oncategory" value="1" class="checkbox show_specific" 
+                                            <?php echo ($checked)?'checked':'';?> />
+                                        <label class="option" for="show_oncategory">Show on taxonomy pages</label>
+                                        <div class="description">
+                                            Select to show on pages for taxonomy terms
+                                        </div>										
+                                    </div>
+                                    <div class="form-item form-type-textarea div_show_specific">
+                                        <label for="hide_oncustom">Show on pages:</label>
+                                        <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea">
+                                            <textarea class="form-textarea show_specific" name="show_oncustom" id="show_oncustom" cols="40" rows="6"><?php if (isset($display_opts->show_oncustom) && !empty($display_opts->show_oncustom)) {
+                                                $whitelist = json_decode($display_opts->show_oncustom);                                             
+                                                foreach ($whitelist as $page) { echo $page."\r\n"; }
+                                            } else {
+                                            } ?></textarea>
+                                            <div class="grippie"></div>
```

Comment:
The grippie doesn't work so it can be removed if not needed. It might confuse people that try to drag it down.

Reply:
Removed.  I saw it work on another admin page and thought it was a good idea but the bit of JS powering it does not seem to be universal